aws/resource_aws_rds_cluster.go

```diff
@@ -222,6 +224,29 @@ func resourceAwsRDSCluster() *schema.Resource {
 				Optional: true,
 			},
 
+			"iam_role": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"role_arn": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"role_status": {
+							Type:     schema.TypeString,
+							Computed: true,
```

Comment:
@radeksimko / @apparentlymart back in hashicorp/terraform#10822 I essentially suggested this inclusion. It would seem I didn't do my homework enough and I now think I should not have. 

I'll take your suggestion and rework this _back_ to being a set of strings, and add another param to expose the statuses, if we're all on that page?